### PR TITLE
const_eval_select: add tracking issue

### PR DESCRIPTION
**Pull request by RalfJung**

### CODE REVIEW

File: library/core/src/intrinsics.rs [hunk 1]
```diff
@@ -2578,8 +2578,7 @@ extern "rust-intrinsic" {
 ///
 /// Currently such an assertion would always succeed; until Rust decides
 /// otherwise, that principle should not be violated.
-#[rustc_const_unstable(feature = "const_eval_select", issue = "none")]
-#[unstable(feature = "core_intrinsics", issue = "none")]
```
Comment (compiler-errors) [inline library/core/src/intrinsics.rs:2582]: don't we need the non-const stability too?

Comment (RalfJung) [reply library/core/src/intrinsics.rs:2582]: Everything in this module is implicitly unstable under core_intrinsics, I think. Most of the other intrinsics here do not have that attribute either, no idea what it is present for some of them.

**Comment by joboet** [comment]
LGTM, r=me once CI is green

**Comment by RalfJung** [comment]
Uh, somehow this is still in the bors queue...
@bors r-

**Pull request merged**